assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -200,35 +200,33 @@ const RoutePatternLayers = ({
   isSelected: boolean
 }): JSX.Element => {
   return routePattern.shape ? (
-    <>
-      <ZoomLevelWrapper>
-        {(zoomLevel) => {
-          return (
-            <>
-              {routePattern.shape && (
-                <>
-                  <RouteShape
-                    shape={routePattern.shape}
-                    isSelected={isSelected}
+    <ZoomLevelWrapper>
+      {(zoomLevel) => {
+        return (
+          <>
+            {routePattern.shape && (
```

Comment:
Is this necessary? Don't we have the guarantee that `routePattern.shape` will be truthy because of the ternary on line 202?

Reply:
That's true, but unfortunately (I think because this use of `routePattern.shape` being wrapped inside of a closure), TypeScript doesn't realize that and throws an error without the check in place.